AlyxToNWB/alyx_to_nwb_metadata.py

```diff
@@ -438,7 +495,7 @@ def behavior_metadata(self):
         behavior_objects = ['wheel', 'wheelMoves', 'licks', 'lickPiezo', 'face', 'eye']
         current_behavior_objects = self._get_current_object_names(behavior_objects)
         for k, u in enumerate(current_behavior_objects):
```

Comment:
I don't really like single-letter variable names. I would use key, value if you can't think of better names